### PR TITLE
Documentation Rehaul

### LINKED ISSUE #283: OAuth API Scopes - User Improvement
In the README file, we have a list of OAuth API scopes that need to be enabled. The list is a bulleted list. Google takes a comma delimited list. 

It might slightly improve user experience to be able to cut and paste this list easily.

### LINKED ISSUE #306: Split up the main README
# 💡 Summary #
Split up the main README into separate files.

## Motivation and context ##
The README is growing large and unwieldy. Splitting it into smaller files will improve readability and maintainability.

## Implementation notes ##
See ScubaGear's README for reference: https://github.com/cisagov/ScubaGear/tree/main

## Acceptance criteria ##
- [ ] The README has been split into different files, using a structure similar to ScubaGear's documentation.

### LINKED ISSUE #314: Rename the baseline markdown files
# 💡 Summary #
Rename the baseline markdown files.

## Motivation and context ##
- The term "minimum viable" has been removed from baseline titles
- Consistency with ScubaGear
- Version numbers in the file names will eventually become problematic once each control has its own version number
- As is, the titles are so verbose it can be hard to immediately see from a glance what the product in question is.
- More readable URLs (e.g., "https://github.com/cisagov/ScubaGoggles/blob/main/baselines/chat.md" instead of "https://github.com/cisagov/ScubaGoggles/blob/main/baselines/Google%20Chat%20Minimum%20Viable%20Secure%20Configuration%20Baseline%20v0.2.md")

## Implementation notes ##
Recommended file name changes:
- Common Controls Minimum Viable Secure Configuration Baseline v0.2.md -> commoncontrols.md
- Gmail Minimum Viable Secure Configuration Baseline v0.2.md -> gmail.md
- Google Calendar Minimum Viable Secure Configuration Baseline v0.2.md -> calendar.md
- Google Chat Minimum Viable Secure Configuration Baseline v0.2.md -> chat.md
- Google Classroom Minimum Viable Secure Configuration Baseline v0.2.md -> classroom.md
- Google Drive and Docs Minimum Viable Secure Configuration Baseline v0.2.md -> drivedocs.md
- Google Meet Minimum Viable Secure Configuration Baseline v0.2.md -> meet.md
- Google Sites Minimum Viable Secure Configuration Baseline v0.2.md -> sites.md
- Groups for Business Minimum Viable Secure Configuration Baseline v0.2.md -> groups.md

## Acceptance criteria ##
- [ ] The markdown file names are simplified
- [ ] ScubaGoggles has been tested and still works as expected

### PR DESCRIPTION
**Pull request by adhilto**
## 🗣 Description ##
- Splits up the main README file into separate files, similar to the ScubaGear README 
- Change the OAuth scope list into a comma-separated list

### 💭 Motivation and context
Closes #306.
Closes #283.

## 🧪 Testing
I clicked all the links in the new documentation files to ensure none were broken.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] If applicable, *All* future TODOs are captured in issues, which are referenced in the PR description.
- [x] The relevant issues PR resolves are linked preferably via [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [x] All relevant type-of-change labels have been added.
- [x] I have read and agree to the [CONTRIBUTING.md](https://github.com/cisagov/ScubaGoggles/blob/main/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge Checklist

- [ ] This PR has been smoke tested to ensure main is in a functional state when this PR is merged.
- [ ] Squash all commits into one PR level commit using the `Squash and merge` button.

## ✅ Post-merge Checklist

- [ ] Delete the branch to clean up.
- [ ] Close issues resolved by this PR if the [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) did not activate.

### CODE REVIEW

File: docs/installation/DownloadAndInstall.md
Comment (mdueltgen) [inline docs/installation/DownloadAndInstall.md:1]: In a pilot meeting we had an issue with Python versions. @rlxdev do you remember what version we had them install? I remember this being a point we had made to update in the readme.

**Review by mdueltgen — COMMENTED** [review]
I left a comment about Python version requirements. I thought we needed something like 3.11 now for some of the new code. We ran into an issue with 3.7 in a pilot meeting maybe @LaurenBassett @rlxdev would remember the version.

**Comment by rlxdev** [comment]
Hi Max (@mdueltgen), During the NARA pilot meeting, the person from NARA trying to get Scubagoggles installed unsuccessfully  tried Python 3.7.* (whatever our minimum is).  The problem was that the specific version of the pandas package (as defined in requirements.txt) was not available for the Python version.  Python version 3.7.* is obsolete (same is true really for Python 3.8 and 3.9 (maybe 3.10 as well)), as the current Python version is 3.12).  I don't really remember what version that was installed in that meeting to make progress, but I'm sure it was at least 3.9.

File: docs/installation/DownloadAndInstall.md
Comment (adhilto) [reply docs/installation/DownloadAndInstall.md:1]: Thanks for bringing this up, I wasn't aware of this issue. Looks like need to refresh our dependencies. I do think that's out of the scope of this PR though as that would involve significant testing and other Python dependencies need to be refreshed. For example, I spent a bit of time today looking into this and while doing so I noticed that the `requests` version we use has been [yanked](https://pypi.org/project/requests/#history). I will open a new issue so we can track those items separately.

Comment (adhilto) [reply docs/installation/DownloadAndInstall.md:1]: https://github.com/cisagov/ScubaGoggles/issues/321

Comment (mdueltgen) [reply docs/installation/DownloadAndInstall.md:1]: Got it, will review/approve now

**Review by mdueltgen — APPROVED** [review]

**Review by mitchelbaker-cisa — APPROVED** [review]

**Comment by adhilto** [comment]
Quick heads up @mdueltgen @mitchelbaker-cisa, I'm going to be reverting the baseline markdown file name changes per David's recommendation so as to not cause merge conflicts with the other ~10 open PRs.

**Pull request merged**